src/interfaces/IProposalTypesConfigurator.sol

```diff
@@ -48,11 +48,24 @@ interface IProposalTypesConfigurator {
         GREATER_THAN
     }
 
+    enum SupportedTypes {
+        UINT8,
+        UINT16,
+        UINT32,
+        UINT64,
+        UINT128,
+        UINT256,
+        ADDRESS,
+        BYTES32,
+        NONE
```

Comment:
is `NONE` needed? if yes, should go on top before UINT8 to be the default one

Reply:
Maybe not if we are going to just revert on unsupported types. I'm not sure if this is intended behavior, because we might not want to revert on a scenario where a scope exists for a function that has a mixture of supported/unsupported types